### ISSUE TITLE
Tooltips provided for the controls present in the "Resource" page are not available while navigating using keyboard TAB/ arrow keys: A11y_.NetAspireDashboard_Resource_Resource_keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/aspire#3174]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Tags:
#A11yTCS; #A11yMAS; #A11ySev3; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. Tab through the page and check if tooltips are available while navigating using keyboard.

### Actual Results:
Tooltips provided for the controls present in the "Resource" page are not available while navigating using keyboard TAB/ arrow keys.

**Similar issue is also observed in below scenario:
Issue 1: Resources_ Details view**

1. Hit the URL: https://localhost:15887 and login with valid credentials.
2. TAB to "Resources" tab item and hit ENTER key.
3. TAB to "View" button present under 'Details' column header in the page and hit ENTER key.
4. "Details container" will open.
5. TAB through the page and check if tooltips are available while navigating using keyboard.

### Expected Results:
Tooltips provided for the controls present in the "Resource" page should be accessible while navigating using keyboard TAB/ arrow keys.

### User Impact:
Keyboard users will not get the information about the control as tooltip is not accessible using keyboard keys.

### Attachment:
[3174_A11y_.NetAspireDashboard_Resource_tooltips_keyboard.webm](https://github.com/dotnet/aspire/assets/90901519/37ce9624-4fdd-4af0-a21b-2a4c18bc839c)

![Resource_DetailsView_Tooltip](https://github.com/dotnet/aspire/assets/90901519/539d0d02-6d02-48c2-a0a3-8141a8189931)

**Comment by adamint** [comment]
@JamesNK this is what I mean by `title` not being accessible. We should convert them into `FluentTooltip`

**Comment by JamesNK** [comment]
Is there a performance overhead for using FluentTooltip? I worry about having dozens or hundreds of them on a page.

**Comment by adamint** [comment]
I'm not sure how they work internally, @vnbaaij?

**Comment by vnbaaij** [comment]
The `FluentTooltip` is a very thin wrapper around the `<fluent-tooltip` web component. We just pass through the parameters.

I did a small test in which I added `role="tooltip"` to the web component and that lit up that the text in the tooltip will be read by a screenreader (tested with NVDA) when the mouse hovers over the tooltip. I will add that for v4.10.2 (or 4.11, whichever version comes next)

**Comment by adamint** [comment]
I can still reproduce the keyboard tooltip gap. Edge 151 at 1280x720 on daily 13.5 preview: I tabbed through the Resources header buttons, cells, resource action buttons, and header buttons. Focused elements had `title`/`aria-label`, but no `fluent-tooltip`/`role=tooltip` appeared after focus waits. I did not run NVDA/Windows, so this is keyboard + DOM evidence only.
